### PR TITLE
fix: `--comfyui-listen` is str

### LINKED ISSUE #27: Tab is empty when using headless server
I have Automatic1111 running on a remote server in the building, accessible on the local network. That UI loads as expected. I can start main.py in ComfyUI with --listen [address] and then open the ComfyUI remotely as expected. I have added the path and necessary switch --listen [address] to the settings for the plugin and reloaded with successful console:

Launching ComfyUI with arguments: --listen [address] --port 8188
Total VRAM reported here
xformers version: 0.0.20
Set vram state to: NORMAL_VRAM
Device: cuda:0 GPU here
Using xformers cross attention
Starting server

To see the GUI go to: http://192.168.0.51:8188

and in this state I can see both the Automatic UI, and the ComfyUI on their relevant remote ports.

Clicking on the tab does not load the interface in either firefox or Edge.

### PR DESCRIPTION
**Pull request by ljleb**
closes #27

### CODE REVIEW

**Comment by ljleb** [comment]
manual test with --listen 0.0.0.0 seems to work

**Comment by ljleb** [comment]
Should we use the url of --listen here?

https://github.com/ModelSurge/sd-webui-comfyui/blob/bd1b40c0c7e195a105ef27938cc31165694b2647/lib_comfyui/webui_tab.py#L70

**Comment by John-WL** [comment]
> Should we use the url of --listen here?
> 
> https://github.com/ModelSurge/sd-webui-comfyui/blob/bd1b40c0c7e195a105ef27938cc31165694b2647/lib_comfyui/webui_tab.py#L70

Yeah I think so

**Review by John-WL — APPROVED** [review]

**Comment by John-WL** [comment]
tests running for 3h straight sounds like a red flag

**Comment by ljleb** [comment]
Doesn't seem to work if it's not `127.0.0.1`. I don't think it makes sense to put any other value, but if it does for some reason hopefully someone raises an issue about this.

**Comment by ljleb** [comment]
> tests running for 3h straight sounds like a red flag

We need to update the testing pipeline to use webui==1.4.0

File: lib_comfyui/webui_settings.py [hunk 1]
```diff
@@ -9,7 +12,10 @@ def create_section():
     shared.opts.add_option("comfyui_install_location", shared.OptionInfo(
         install_comfyui.default_install_location, "ComfyUI install location", section=section))
     shared.opts.add_option("comfyui_additional_args", shared.OptionInfo(
-        '', "Additional cli arguments to pass to ComfyUI (requires reload UI)", section=section))
+        '', "Additional cli arguments to pass to ComfyUI (requires reload UI. Do NOT prepend --comfyui-, these are literally forwarded to comfyui)", section=section))
```
Comment (John-WL) [inline lib_comfyui/webui_settings.py:15]: can we change "literally" for "directly"?

**Review by John-WL — APPROVED** [review]

**Review by John-WL — APPROVED** [review]

**Comment by John-WL** [comment]
> > tests running for 3h straight sounds like a red flag
> 
> We need to update the testing pipeline to use webui==1.4.0

how do we do this

**Comment by ljleb** [comment]
Need to look at the workflow setup in a1111

**Pull request merged**